doc/client_lua_api.md

```diff
@@ -653,7 +611,7 @@ Helper functions
 * `table.copy(table)`: returns a table
     * returns a deep copy of `table`
 
-`minetest` namespace reference
+Minetest namespace reference
```

Comment:
You can use `minetest` since `minetest` is declared in https://github.com/minetest/minetest/blob/master/builtin/init.lua#L25 and that code is ran no matter which scripting interface is being started. I reason I changed `minetest` to Minetest  is because it looks nice after rendering to html.